### PR TITLE
Sloped tiles in runways and `canShineCharge`

### PR DESCRIPTION
**Pull request by fdtaillefer**

### CODE REVIEW

File: region/brinstar/pink.json [hunk 1]
```diff
@@ -1410,9 +1412,19 @@
...
                 "requires": null,
                 "openEnd": 0
               }
+            ],
+            "canLeaveCharged": [
+              {
+                "usedTiles": 21,
+                "steepUpTiles": 2,
+                "framesRemaining": 0,
+                "shinesparkFrames": 20,
```
Comment (DIProgan) [inline region/brinstar/pink.json]: Nice one

File: region/crateria/east.json [hunk 2]
```diff
@@ -972,9 +973,17 @@
...
+                "steepUpTiles": 1,
                 "requires": null,
                 "openEnd": 0
+              },
+              {
+                "length": 28,
+                "steepUpTiles": 5,
+                "steepDownTiles": 1,
+                "requires": ["Gravity"],
+                "openEnd": 0
```
Comment (DIProgan) [inline region/crateria/east.json]: This one also needs the usableComingIn false because of Sciser hits

File: region/norfair/east.json [hunk 3]
```diff
@@ -3840,9 +3849,14 @@
...
-                  {"heatFrames": 90}
+                  {"heatFrames": 90},
+                  {"resetRoom":{
+                    "nodes": [2],
+                    "mustStayPut": true
+                  }}
                 ],
-                "openEnd": 1
+                "openEnd": 1,
+                "note": "Can't charge in lava, so doesn't work coming in from the top left door"
```
Comment (DIProgan) [inline region/norfair/east.json]: Unless you have Gravity

Comment (fdtaillefer) [reply region/norfair/east.json]: No, you can't charge in lava. Gravity allows you to charge in acid.

File: region/crateria/east.json (hunk 2 above)
Comment (fdtaillefer) [reply region/crateria/east.json]: I neglected to actually check that room, I think. I'll go take a look and adjust.

File: region/norfair/east.json (hunk 3 above)
Comment (DIProgan) [reply region/norfair/east.json]: Weird TIL

**Comment by DIProgan** [comment]
Thumbs up

**Pull request merged**